### PR TITLE
Add support uploading files from url

### PR DESCRIPTION
**Pull request by bautrukevich**

### CODE REVIEW

File: src/fileFrom/UploadFrom.ts [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+    value: 0,
+  }
+  protected file: UploadcareFile | undefined
+  onProgress: ((progress: UploadingProgress) => void) | null = null
+  onUploaded: Function | null = null
+  onReady: Function | null = null
+  onCancel: Function | null = null
+
+  protected setProgress(state: ProgressState, progressEvent?: ProgressEvent) {
+    switch (state) {
```
Comment (nd0ut) [inline src/fileFrom/UploadFrom.ts]: Я б этот свич отрефакторил, какой-то он длинный и повторяющийся

**Comment by bautrukevich** [comment]
[UploadClient: add the support of url uploads in fileFrom](https://trello.com/c/ysIyNMvv/70-uploadclient-add-the-support-of-url-uploads-in-filefrom)

File: src/fileFrom/UploadFrom.ts (hunk 1 above)
Comment (bautrukevich) [reply src/fileFrom/UploadFrom.ts]: Ok, thank you! I will try : )

**Comment by bautrukevich** [comment]
[UploadClient: fix progress of fileFrom](https://trello.com/c/vJcdCzfE/127-uploadclient-fix-progress-of-filefrom)

**Pull request merged**